src/toil/test/cwl/cwlTest.py

```diff
@@ -631,6 +639,162 @@ def test_kubernetes_cwl_conformance(self, **kwargs):
     def test_kubernetes_cwl_conformance_with_caching(self):
         return self.test_kubernetes_cwl_conformance(caching=True)
 
+@needs_aws_ec2
+@needs_fetchable_appliance
+@slow
+class CWLOnARMTest(ToilTest):
+    def __init__(self, methodName):
+        super().__init__(methodName=methodName)
+        self.keyName = os.environ.get('TOIL_AWS_KEYNAME', 'id_rsa')
+        self.instanceTypes = ["t4g.large"]
+        self.clusterName = 'cwl-test-' + str(uuid.uuid4())
+        self.numWorkers = ['2']
+        self.numSamples = 2
+        self.spotBid = 0.15
+        self.zone = get_best_aws_zone()
+        assert self.zone is not None, "Could not determine AWS availability zone to test in; is TOIL_AWS_ZONE set?"
+        # We need a boto2 connection to EC2 to check on the cluster
+        self.boto2_ec2 = boto.ec2.connect_to_region(zone_to_region(self.zone))
+
+        # Where should we put our virtualenv?
+        self.venvDir = '/tmp/venv'
+        # Where should we put our data to work on?
+        # Must exist in the Toil container; the leader will try to rsync to it
+        # (for the SSE key) and not create it.
+        self.dataDir = '/tmp'
```

Comment:
Some of these fields (`dataDir`, for example) aren't used in this new copy and should be removed.